.github/workflows/pr_checks.yml

```diff
@@ -0,0 +1,31 @@
+name: PR Checks
```

Comment:
I've added this basic github actions workflow to ensure pre-commit and pytest can run successfully before merging.

But they don't run when I push to this PR. Do you need to enable github actions?

Reply:
Has to be merged into main first I think.